File: RelationExample/RelationExample/CheckingAccount.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RelationExample
{
    class CheckingAccount
    {
        private int accNumber;

        public int GetAccNumber()
        {
            return accNumber;
        }
        public void SetAccNumber(int number)
        {
            accNumber = number;
        }

        public CheckingAccount(int accNumber)
        {
            this.accNumber = accNumber;
        }


    }
}

File: RelationExampleC++/RelationExampleC++/CheckingAccount.cpp
#include "CheckingAccount.h"


CheckingAccount::CheckingAccount(int accNumber) : m_accNumber(accNumber)
{}

int CheckingAccount::GetAccNumber()
{
	return m_accNumber;
}

void CheckingAccount::SetAccNumber(int number)
{
	m_accNumber = number;
}